mir-module/src/main/resources/xsl/response-facets.xsl

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:encoder="xalan://java.net.URLEncoder"
+                xmlns:i18n="xalan://org.mycore.services.i18n.MCRTranslation"
+                xmlns:mcrxsl="xalan://org.mycore.common.xml.MCRXMLFunctions"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                exclude-result-prefixes="i18n mcrxsl encoder xalan fn">
+
+  <xsl:param name="MIR.Response.Facet.Prefix.Classification"/>
+
+  <xsl:template name="facets">
+
+    <xsl:for-each select="/response/lst[@name='facet_counts']/lst[@name='facet_fields']/*">
+      <xsl:variable name="facet_name" select="self::node()/@name"/>
+      <xsl:variable name="classId">
+        <!-- Check if facet has a prefix -->
+        <xsl:choose>
+          <xsl:when test="contains($facet_name, $MIR.Response.Facet.Prefix.Classification)">
```

Comment:
this property is relevant at index time but not at search time thus can be removed

Reply:
fixed